docs/admin/metrics/eventing-metrics-api.md

```diff
@@ -1,19 +1,16 @@
 ---
-title: "Metrics API"
-weight: 99
+title: "Knative Eventing metrics API"
+weight: 10
 type: "docs"
+aliases:
+  - /docs/eventing/metrics
 ---
 
-<br>
+Administrators can monitor the Knative Eventing control plane by looking at the metrics exposed by each Knative Eventing component.
```

Comment:
I think it would be more data plane?

Reply:
Mention both in next commit, see below comment re follow up